<launch>
    <node pkg="waiter_ros" type="robot_arm_interface.py" name="wheel" output="screen"></node>
    <node pkg="waiter_ros" type=".py" name="diff_tf" output="screen"></node>
</launch>